packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -130,6 +180,258 @@ class _SystemContextMenuState extends State<SystemContextMenu> {
   @override
   Widget build(BuildContext context) {
     assert(SystemContextMenu.isSupported(context));
+    if (isFirstBuild) {
+      isFirstBuild = false;
+      _show();
+    }
+
     return const SizedBox.shrink();
   }
 }
+
+// TODO(justinmc): Still need to decide to name this as ios specific or not.
+// Probably should make it ios-specific, because these classes encode
+// ios-specific logic, such as the fact that you can't change the title of the
+// paste button, etc.
```

Comment:
Alternatively, maybe always pass the title to engine for copy/paste/cut as well? It's just that engine will just ignore that. Will this allow you to share the types across platforms?